CHANGES/23.feature

```diff
@@ -0,0 +1 @@
+Add content sub command
```

Comment:
Let's keep this as the only changelog entry for this change, but extend it as follows:
```suggestion
Added the content sub command, which provides upload and ??? functionality.
```
Substitute for my ??? as appropriate, I have not yet looked at the details of what is actually being added. :wink:

Reply:
I'm going to suggest that 20 be the driving change as it is the earliest and that 23 be marked as duplicate of 20.